Firestore/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Unreleased
 
+# v0.10.2
+- [changed] When you delete a FirebaseApp, the associated Firestore instances
+  are now also deleted (#683).
+- [fixed] Fixed race conditions in streams that could be exposed by rapidly
+  toggling the network from enabled to disabled and back (#772) or encountering
+  multiple successive failures from the server (#835).
```

Comment:
It actually only takes a single failure (it happens if you do 2+ writes without waiting for acknowledgement and the first one fails), so to be completely accurate, I think you should change "multiple successive failures" to "a failure"